packages/block-library/src/query/utils.js

```diff
@@ -375,7 +379,23 @@ export const useUnsupportedBlocks = ( clientId ) => {
 			getClientIdsOfDescendants( clientId ).forEach(
 				( descendantClientId ) => {
 					const blockName = getBlockName( descendantClientId );
-					if ( ! blockName.startsWith( 'core/' ) ) {
+					/*
+					 * Client side navigation can be true in two states:
+					 *  - supports.interactivity = true;
+					 *  - supports.interactivity.clientNavigation = true;
+					 */
+					const blockInteractivityBool =
+						getBlockSupport( blockName, 'interactivity' ) === true;
+					const blockClientNavigation = getBlockSupport(
+						blockName,
+						'interactivity.clientNavigation'
+					);
+					const blockInteractivity =
+						blockInteractivityBool || blockClientNavigation;
+					if (
+						! blockName.startsWith( 'core/' ) &&
```

Comment:
I don’t think you need to check against core blocks anymore now that block support allows opting into client-side navigation.